index.html

```diff
@@ -48,7 +48,7 @@ <h1>Tic Tac Toe</h1>
 </table>
 
 <h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
-You could move the scripts tags to the end of the document
+
```

Comment:
```suggestion
<p hidden>Thank you for playing our game</p>
```

Reply:
I do like this suggestion to thank the game player.